src/components/CommunityEvents.tsx

```diff
@@ -0,0 +1,256 @@
+            title: event.summary,
+            calendarLink: event.htmlLink,
+            pastEventLink: event.location,
+          }
+        })
+        setState({
+          ...state,
+          pastEventData,
+          upcomingEventData,
+          loading: false,
+          hasError: false,
+        })
+      }
+      fetchCalendarData()
+    } catch {
+      setState({ ...state, loading: false, hasError: true })
+    }
+  }, [])
+
+  const renderEventDateTime = (
+    date,
+    language,
+    params = {
+      year: "numeric",
+      month: "long",
+      day: "numeric",
+      hour12: false,
+      hour: "numeric",
+      minute: "numeric",
+    }
+  ) => {
+    return DateTime.fromISO(date).setLocale(language).toLocaleString(params)
+  }
+
+  const renderEventLink = (link, title) => {
+    return (
+      <Link to={link} fontWeight="700">
+        {title}
+      </Link>
+    )
+  }
+
+  const renderEvent = (event, language) => {
+    const { date, title, calendarLink } = event
+    const params = {
+      year: "numeric",
+      month: "short",
+      day: "numeric",
+    }
+    return (
+      <Flex gap={6}>
+        <Text>{renderEventDateTime(date, language, params)}</Text>
+        {renderEventLink(calendarLink, title)}
+      </Flex>
+    )
+  }
+
+  return (
+    <Flex w="full" flexDirection={{ base: "column", lg: "row" }}>
+      <Center w={{ base: "100%", lg: "40%" }} px={16}>
+        <Box>
+          <Heading>
+            <Translation id="community-events-content-heading" />
+          </Heading>
+          <Text>
+            <Translation id="community-events-content-1" />
+          </Text>
+          <Text>
+            <Translation id="community-events-content-2" />
+          </Text>
+        </Box>
+      </Center>
+      <Flex
+        w={{ base: "100%", lg: "60%" }}
+        flexDirection={{ base: "column", lg: "row" }}
+      >
+        <Flex
+          w={{ base: "100%", lg: "50%" }}
+          bg="layer2Gradient"
+          p={12}
+          textAlign="center"
+          flexDir="column"
+        >
+          <Text fontSize="md" fontWeight="bold">
+            <Translation id="community-events-next-event" />
+          </Text>
+          {state.loading ? (
+            <Text>
+              <Translation id="loading" />
+            </Text>
+          ) : (
+            <Box>
+              {state.hasError ? (
+                <Text color="error">
+                  <Translation id="loading-error-try-again-later" />
+                </Text>
+              ) : state.upcomingEventData.length ? (
+                <>
+                  <Text m={0} fontSize="xl">
+                    {renderEventDateTime(
+                      state.upcomingEventData[0].date,
+                      language
+                    )}
+                  </Text>
+                  <Text color={"bodyLight"} fontSize="md">
+                    ({Intl.DateTimeFormat().resolvedOptions().timeZone})
+                  </Text>
```

Comment:
Should be 16px between timezone and event title

<img width="448" alt="Screenshot 2023-06-08 at 12 32 19" src="https://github.com/ethereum/ethereum-org-website/assets/62268199/b3a8d443-209b-4112-bb09-4c96a661c69d">

```suggestion
                  <Text mb={1} color={"bodyLight"} fontSize="md">
                    ({Intl.DateTimeFormat().resolvedOptions().timeZone})
                  </Text>
```